cms/djangoapps/contentstore/tasks.py

```diff
@@ -14,6 +15,11 @@
 from contentstore.utils import initialize_permissions
 from opaque_keys.edx.keys import CourseKey
 
+from contentstore.courseware_index import CoursewareSearchIndexer, SearchIndexingError
+
+LOGGER = get_task_logger(__name__)
```

Comment:
@martynjames nit: correct me if I'm wrong, but logger here is not a constant, and all caps are reserved for constants. It's not modified within the module, so it essentially is a constant, but it's more like a module memeber as it's instantiated as a result of function call. Also (primary argument), I haven't seen module-level loggers in all caps anywhere in the code body.